ocs_ci/helpers/dr_helpers.py

```
@@ -1699,59 +1753,78 @@ def verify_last_group_sync_time(
+    with config.RunWithAcmConfigContext():
+        previous_last_group_sync_time = initial_last_group_sync_time
+        last_group_sync_time = None
+
+        for iteration in range(1, num_iterations + 1):
+            wait_for_set = previous_last_group_sync_time is None
+            if wait_for_set:
+                logger.info("Waiting for lastGroupSyncTime to be set")
+            elif iteration > 1:
+                logger.info(
+                    f"Waiting for lastGroupSyncTime change "
+                    f"(iteration {iteration}/{num_iterations})"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)